apps/mobile-wallet/src/store/addresses/addressesSelectors.ts

```diff
@@ -23,24 +23,6 @@ import { createSelector } from '@reduxjs/toolkit'
 import { makeSelectAddressesKnownFungibleTokens, selectAllAddresses } from '~/store/addressesSlice'
 import { RootState } from '~/store/store'
 
-// TODO: Same as in desktop wallet
-export const selectHaveHistoricBalancesLoaded = createSelector(selectAllAddresses, (addresses) =>
-  addresses.every((address) => address.balanceHistoryInitialized)
-)
-
-export const selectAddressesHaveHistoricBalances = createSelector(
-  selectAllAddresses,
-  (addresses) =>
-    addresses.every((address) => address.balanceHistoryInitialized) &&
-    addresses.some((address) => address.balanceHistory.ids.length > 0)
-)
-
-// TODO: Same as in desktop wallet
-export const selectIsStateUninitialized = createSelector(
-  (state: RootState) => state.addresses.status,
-  (status) => status === 'uninitialized'
-)
-
```

Comment:
🗑️ not needed after the code simplification!

Reply:
Feels good.